src/components/WizardModal/WizardModal.tsx

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import styled from 'styled-components';
+import usePortal from 'react-cool-portal';
+import PropTypes from 'prop-types';
+
+import { getColor, getDepth, getRadii, pxToRem } from '../../utils';
+import { ModalSizes } from '../Modal/Modal.enums';
+import { WizardModalProps } from './WizardModal.types';
+import { Padbox } from '../layout';
+
+const widthVariants = {
+  [ModalSizes.xs]: 320,
+  [ModalSizes.sm]: 430,
+  [ModalSizes.md]: 600,
+  [ModalSizes.lg]: 900,
+};
```

Comment:
I personally think that this modal definitely should not exist in `xs` and `sm` sizes, we can discuss the `md` (but I would remove that also). 
Please follow Figma discussion